apiconnector/src/main/java/org/openml/apiconnector/xml/DataSetDescription.java

```diff
@@ -86,7 +86,7 @@
 	@XStreamAlias("oml:row_id_attribute")
 	private String row_id_attribute;
 
-	@XStreamImplicit(itemFieldName = "ignore_attribute")
+	@XStreamImplicit(itemFieldName = "oml:ignore_attribute")
```

Comment:
would you mind also extending a unit test (e.g., one that downloads a dataset with ignore attribute from the test set, and checks whether it is not empty)?

Thanks!